src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.hpp

```diff
@@ -177,24 +181,72 @@ class ReconstructionEngine_sequentialSfM : public ReconstructionEngine
    * @param[in] remainingViewIds: input list of remaining view IDs in which we will search for the best ones for resectioning.
    * @return False if there is no possible resection.
    */
-  bool FindNextImagesGroupForResection(
+  bool findNextImagesGroupForResection(
     std::vector<size_t>& out_selectedViewIds,
     const std::set<size_t>& remainingViewIds) const;
 
+  struct ResectionData : ImageLocalizerMatchData
+  {
+    std::set<std::size_t> tracksId; /// tracks index for resection
```

Comment:
`///` comments are applied on the next line:
```
/// comment related to b
int b;
```
So here it will be interpreted the same:
```
int a; /// comment related to b
int b;
```
You can use `//<` for inline doc or use a new line:
```
int a; //< inline comment related to a
/// comment related to b
int b;
```